### ISSUE TITLE
Web-UI

### ISSUE THREAD

**Issue by lquenti** [lquenti/ankiding#14]
Only after v1 is published; unclear whether we try the cool FE-only version (webassembly pipeline) or rely on a traditional API

**Comment by lquenti** [comment]
If we do BE only, we should de-couple the rust code from the FE, especially since we only need a static form that sends the stuff towards a HTTP API

**Comment by lquenti** [comment]
I recently saw (at cloudflare) that browsers support directory upload!

This means that there is literally no reason to not mainly target web. 

Of couse, we should implement the desktop UI first as we then do not have to think about whether all crates are compatible with the webassembly target...

Next, we also should implement a "dont download any files" flag first as CORS will otherwise break everything.

**Comment by lquenti** [comment]
this is so far in the future, maybe in a new issue. For now, #59 is the successor.
